annotation-utils/src/main/java/eu/europeana/annotation/config/SubtitleFormats.java

```diff
@@ -0,0 +1,29 @@
+package eu.europeana.annotation.config;
+
+import java.util.List;
+import java.util.Optional;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+@JacksonXmlRootElement(localName = "subtitle-formats")
+public class SubtitleFormats {
```

Comment:
this is a data model class, please follow the conventiosn and move it to .model package

Reply:
Done.